iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningServiceClientTests.java

```diff
@@ -86,4 +90,57 @@ public void enrollmentGroupGetAttestationMechanismSymmetricKey() throws Provisio
         assertTrue(retrievedSymmetricKeyAttestation.getPrimaryKey().length() > 0);
         assertTrue(retrievedSymmetricKeyAttestation.getSecondaryKey().length() > 0);
     }
+
+    @Test
+    public void createdEnrollmentGroupMatchesSentEnrollmentGroup() throws ProvisioningServiceClientException
+    {
+        String groupId = testPrefix + UUID.randomUUID();
+        Attestation attestation = new SymmetricKeyAttestation("", "");
+        EnrollmentGroup enrollment = new EnrollmentGroup(groupId, attestation);
+        enrollment.setAllocationPolicy(AllocationPolicy.CUSTOM);
+        CustomAllocationDefinition customAllocationDefinition = new CustomAllocationDefinition();
+        customAllocationDefinition.setApiVersion("2018-09-01-preview");
```

Comment:
is this something users need to be aware of or set?

Reply:
Yes, albeit I had a hard time finding the expected values here